src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -727,24 +794,30 @@ export default class DataTable extends Vue {
   }
 
   serializeData() {
-    const serializeRow = (row: DataTableRow, rowNumber: number, parentRowId: string | null) => {
+    const serializeRow = (
+      row: DataTableRow,
+      rowNumber: number,
+      parent: DataTableRow | undefined,
+      parentRowId: string | null
+    ) => {
       const rowId = this._rowId(row.id || rowNumber);
       const rowN = parentRowId !== null ? `${parentRowId}-${rowNumber}` : String(rowNumber);
       const rowObject = {
         ...row,
+        parent,
         rowNumber: rowN,
-        rowId: rowId,
+        rowId,
       };
       let subrowNumber = 0;
 
       if (
         typeof row.nestedContent === 'object' &&
         typeof row.nestedContent.table === 'object' &&
-        row.nestedContent.table.data
+        !!row.nestedContent.table.data
```

Comment:
Sorry for the confusion..I meant something like this:
!!row.nestedContent && 
typeof row.nestedContent === 'object' &&
typeof row.nestedContent.table === 'object' &&
row.nestedContent.table.data

Reply:
Done!